openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx

```diff
@@ -112,6 +114,47 @@ const ConfigureIngestion = ({
     );
   };
 
+  const [profileSampleError, setProfileSampleError] = useState({
+    profileSample: false,
+  });
```

Comment:
If that just a flag for error can we just have value directly? like this?
```suggestion
  const [profileSampleError, setProfileSampleError] = useState(false);
```